sdk/clientcore/core/README.md

```diff
@@ -1,23 +1,21 @@
 # Core shared library for Java
 
-[![Build Documentation](https://img.shields.io/badge/documentation-published-blue.svg)](https://azure.github.io/azure-sdk-for-java)
+<!-- [![Build Documentation](https://img.shields.io/badge/documentation-published-blue.svg)](https://azure.github.io/azure-sdk-for-java) -->
 
-Core provides shared primitives, abstractions, and helpers for modern Java Core SDK client libraries.
-These libraries follow
-the [SDK Design Guidelines for Java](https://azure.github.io/azure-sdk/java_introduction.html)
-and can be easily identified by package names starting with `io.clientcore` and module names starting with `clientcore`,
-e.g. `io.clientcore.core` would be found within the `/sdk/clientcore/core` directory. A more complete
-list of client libraries using Core can be found [here](https://azure.github.io/azure-sdk/releases/latest/#java-packages).
+Client Core provides shared primitives, abstractions, and helpers for modern SDK client libraries. These libraries
+<!-- follow the [SDK Design Guidelines for Java](https://azure.github.io/azure-sdk/java_introduction.html) and -->
+can be easily identified by package names starting with `io.clientcore` and module names starting with `core`,
```

Comment:
We've changed the pattern where all the Core libraries no longer start with `core`

@JonathanGiles, thoughts on this change?

Reply:
I'll just mention the package names start with `io.clientcore` for the time being.